test/user.test.js

```diff
@@ -214,6 +214,24 @@ describe('User', function() {
       assert(u2.password === u1.password);
     });
 
+    it('invalidates the user\'s accessToken when the user is deleted', function(done) {
+      User.create({ email: 'b@c.com', password: 'bar' }, function(err, instance) {
+        User.login({ email: 'b@c.com', password: 'bar' }, function(err, accessToken) {
+          if (err) return done(err);
+          assert(accessToken.userId);
+          User.deleteById(instance.id, function(err) {
+            if (err) return done(err);
+            AccessToken.findById(accessToken.userId, function(err, notFound) {
```

Comment:
Please add a check that the user instance was deleted. I suspect such check would fail with your current implementation.

```js
User.findById(instance.id, function(err, user) {
  expect(user).to.equal(null);
  AccessToken.findById(...);
});
```